File: algorithms/3280.convert-date-to-binary/solution.go
func convertDateToBinary(date string) string {
	var res []string
	for _, part := range strings.Split(date, "-") {
		num, _ := strconv.Atoi(part)
		res = append(res, fmt.Sprintf("%b", num))
	}
	return strings.Join(res, "-")
}

File: algorithms/3280.convert-date-to-binary/solution.rs
impl Solution {
    pub fn convert_date_to_binary(date: String) -> String {
        let mut res: Vec<String> = vec![];
        for part in date.split("-") {
            res.push(format!("{:b}", part.parse::<i32>().unwrap()));
        }
        res.join("-")
    }
}
